lib/src/chart/base/base_chart/render_base_chart.dart

```diff
@@ -162,6 +167,23 @@ abstract class RenderBaseChart<R extends BaseTouchResponse> extends RenderBox
     }
   }
 
+  /// Invokes the [_touchCallback] to notify listeners of this [FlTouchEvent]
+  ///
+  /// We don't get a response for these events because they are scale events (scroll, pinch.etc)
+  /// We just invoke [_touchCallback] using the [event].
+  void _notifyScaleEvent(FlTouchEvent event) {
+    if (_touchCallback == null) {
+      return;
+    }
+
+    final pointerCount = event.pointerCount;
+
+    R? response;
+    if (pointerCount > 1) {
+      _touchCallback!(event, response);
```

Comment:
You can just set a null as the second parameter (instead of having a nullable R?, which is always null)

Reply:
Will fix.